### ISSUE TITLE
Release vNext

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1223]
Can be labelled **ready** when all other issues on this milestone are closed.

- [ ] ~~run code analysis in VS in *Release* mode and address all coding and style violations (send a regular PR which must be merged before continuing)~~
- [ ] ensure completeness of draft [release notes](https://github.com/FakeItEasy/FakeItEasy/releases), including non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
- change "vNext" names to appropriate x.y.z version for the next:
    - [ ] GitHub Release
    - [ ] milestone
    - [ ] release checklist (this issue)
- create new GitHub artifacts for the next release:
    - [ ] create a new draft GitHub Release with name "vNext"
    - [ ] create a new milestone for the next release with name "vNext"
    - [ ] create a new issue (like this one) with name "Release vNext" for the next release, adding it to the new milestone
- [ ] create release-x.y.z branch on upstream
- [ ] perform any last-minute tasks on release-x.y.z
- [ ] submit a PR to merge the release-x.y.z branch to master
- [ ] submit a PR to merge the release-x.y.z branch to develop, if there are any changes on release-x.y.z
- [ ] once release-x.y.z has been merged, tag master and push to upstream. This will push the nupkgs to [nuget.org](https://www.nuget.org/profiles/FakeItEasy) and attach all nupkgs from the tagged build to the GitHub Release.
- [ ] until #1183 is fixed, make sure the deployment actually succeeded
- [ ] de-list pre-release or superseded buggy NuGet packages if present
- [ ] publish the GitHub release
- [ ] tweet, mentioning contributors, and post link as comment here for easy retweeting ;-)
- [ ] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [ ] post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
- [ ] close this issue
- [ ] close the [milestone](https://github.com/FakeItEasy/FakeItEasy/milestones?state=open) for this release

**Comment by blairconrad** [comment]
Oops.
